concepts.wip/conditionals/introduction.md

```diff
@@ -1,7 +1,39 @@
 # Introduction
 
-There are three primary conditional expressions that are used in Julia:
+Conditional evaluation is one of the handful of ways that Julia offers to _control the flow_ of your program.
+
+There are three primary ways to achieve this in Julia:
 
 - `if`-expressions
 - The ternary operator
 - Short-circuit evaluation
+
+Suppose you need a program to execute a certain set of statements, given that the conditional expression is satisfied, then we structure the code according to the syntax that Julia provides.
+
+Below are examples of the syntax for conditional evaluation in Julia. 
+
+## Examples
+
+```julia
+# Assign x to 3
+julia> x = 3
+3
+
+# if syntax
+julia> if x > 0
+            true
+        else
+            false
+        end
+true
+
+# ternary syntax
+julia> x > 0 ? true : false
+true
+
+# short-circuit syntax
+julia> x > 0 || false
+true
+```
+
+The above code is just to introduce you to the respective syntaxes as defined in Julia. We will treat each of them in detail in their respective section in the Julia track.
```

Comment:
This concept is mainly used for progression/unlocking purposes so that exercise can require knowledge of some form of conditional statement without having to specify if that should be an if statement or short-circuiting. It's unlocked by the concepts that the `about.md` links to. As a result, imho it shouldn't have examples as those are covered in the specific concepts.

Sorry, there's no way you could have known this as it's not properly documented in a findable location.

Reply:
Understood. Thank you for taking the time to explain this. 

I'm assuming that I can provide the appropriate details for ex: in the `if`-expressions section?

Reply:
Yes, in here: https://github.com/exercism/julia/blob/main/concepts.wip/if-expressions/introduction.md